src/ide/vscode/VscodeToggleBreakpoint.ts

```diff
@@ -0,0 +1,60 @@
+import { BreakpointDescriptor } from "@cursorless/common";
+import { toVscodeRange } from "@cursorless/vscode-common";
+import * as vscode from "vscode";
+import { VscodeTextEditorImpl } from "./VscodeTextEditorImpl";
+
+export async function vscodeToggleBreakpoint(
+  editor: VscodeTextEditorImpl,
+  descriptors: BreakpointDescriptor[] | undefined,
+): Promise<void> {
+  if (descriptors == null) {
+    return await vscode.commands.executeCommand(
+      "editor.debug.action.toggleBreakpoint",
+    );
+  }
```

Comment:
note: this seemed like reasonable behaviour when no ranges are passed in.  fwiw, we don't ever utilise this functionality but 🤷‍♂️